Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.8-alpine
```

Comment:
As I already commented in a Pete's PR, maybe it is good to use `golang:1.8.3-alpine` to make it consistent with circle...